integration/helper_test.go

```diff
@@ -433,6 +434,116 @@ options:
 			Expect(err).To(HaveOccurred())
 			assertErrorsHandled()
 		})
+
+		It("skips batches if skip file discovered", func() {
```

Comment:
Is it possible to add a test with non single data file and with "--resize-cluster" option?

Reply:
Test added